beacon-chain/cache/active_count.go

```diff
@@ -0,0 +1,98 @@
+package cache
+
+import (
+	"errors"
+	"strconv"
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"k8s.io/client-go/tools/cache"
+)
+
+var (
+	// ErrNotActiveCountInfo will be returned when a cache object is not a pointer to
+	// a ActiveCountByEpoch struct.
+	ErrNotActiveCountInfo = errors.New("object is not a active count obj")
+
+	// maxActiveCountListSize defines the max number of active count can cache.
+	maxActiveCountListSize = 1000
+
+	// Metrics.
+	activeCountCacheMiss = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "active_count_cache_miss",
```

Comment:
From the metrics page, I think a reader would have a hard time understanding what this metric is. Can you add more to the name or help message?

Reply:
yeah good point, `active_validator_count_cache_miss` is better here